src/apps/sources/__test__/components/SourceForm.test.tsx

```diff
@@ -2,40 +2,37 @@ import React from "react";
 import SourceForm from "../../components/SourceForm";
 import {APIOrg, APIProfile} from "../../../authentication";
 import {Source} from "../../types";
-import {render} from "@testing-library/react";
+import {render, fireEvent, act, wait, waitFor} from "@testing-library/react";
```

Comment:
@binduak @supriyatw can we remove wait and waitFor?

Reply:
Done @vinishad